build.gradle

```diff
@@ -68,19 +69,29 @@ test {
     }
 }
 
-def webViewVersion = '0.1.16'
-def webviewFileName = 'jfrog-ide-webview-' + webViewVersion + '.tgz'
+def webviewFileName = 'jfrog-ide-webview-' + webviewVersion + '.tgz'
+def webviewUrl = 'https://releases.jfrog.io/artifactory/ide-webview-npm/jfrog-ide-webview/-/' + webviewFileName
 tasks.register('downloadWebview', Download) {
-    src 'https://releases.jfrog.io/artifactory/ide-webview-npm/jfrog-ide-webview/-/' + webviewFileName
+    src webviewUrl
     dest buildDir
     onlyIfModified true
+    finalizedBy('getAndUpdateWebviewChecksum')
+}
+
+tasks.register('getAndUpdateWebviewChecksum') {
     finalizedBy('verifyWebview')
+    if (System.getenv("CI") != null) {
+        ext.checksum = webviewChecksum
```

Comment:
Would it be possible to write log message here that the CI mode is active, or something similar?